Microsoft.DotNet.Interactive.Formatting/JsonFormatter.cs

```diff
@@ -25,7 +25,7 @@ static JsonFormatter()
 
         internal static IFormatterSet DefaultFormatters { get; private set; } = new DefaultHtmlFormatterSet();
     
-        internal static JsonSerializerSettings SerializerSettings { get; }
+        public static JsonSerializerSettings SerializerSettings { get; }
```

Comment:
How strict are you about documenting public properties?

You use this in another project, so I would think it'd need some doc comments.

Reply:
We'll do a documentation pass before too long but method signatures might change or get refactored so I prefer to let them stabilize a little first.